DependencyInjection/Configuration.php

```diff
@@ -40,26 +43,91 @@ public function getConfigTreeBuilder() : TreeBuilder
         $organizeMigrationModes = $this->getOrganizeMigrationsModes();
 
         $rootNode
+            ->fixXmlConfig('migration', 'migrations')
+            ->fixXmlConfig('migrations_path', 'migrations_paths')
             ->children()
-                ->scalarNode('dir_name')->defaultValue('%kernel.root_dir%/DoctrineMigrations')->cannotBeEmpty()->end()
-                ->scalarNode('namespace')->defaultValue('Application\Migrations')->cannotBeEmpty()->end()
-                ->scalarNode('table_name')->defaultValue('migration_versions')->cannotBeEmpty()->end()
-                ->scalarNode('column_name')->defaultValue('version')->end()
-                ->scalarNode('column_length')->defaultValue(14)->end()
-                ->scalarNode('executed_at_column_name')->defaultValue('executed_at')->end()
-                ->scalarNode('all_or_nothing')->defaultValue(false)->end()
                 ->scalarNode('name')->defaultValue('Application Migrations')->end()
-                ->scalarNode('custom_template')->defaultValue(null)->end()
-                ->scalarNode('check_database_platform')->defaultValue(true)->end()
-                ->scalarNode('organize_migrations')->defaultValue(false)
+
+                ->arrayNode('migrations_paths')
+                    ->info('A list of namespace/path pairs where to look for migrations.')
+                    ->isRequired()
+                    ->requiresAtLeastOneElement()
+                    ->useAttributeAsKey('namespace')
+                    ->prototype('scalar')->end()
+                 ->end()
```

Comment:
The indentation on this is off:
```suggestion
                ->end()
```